roles/installer/defaults/main.yml

```diff
@@ -188,6 +188,11 @@ session_cookie_secure: False
 
 # Assign a preexisting priority class to the control plane pods
 control_plane_priority_class: ''
+
+redis_resource_requirements:
+  requests:
+    cpu: 500m
```

Comment:
I lowered the minimum requested resources to schedule the redis pod to align with our other pods in a reasonable way.